sdk/src/types/mod.rs

```diff
@@ -74,36 +74,17 @@ impl<'a, T> From<&'a Boo<'a, T>> for Boo<'a, T> {
 #[derive(Clone, Default, Debug)]
 pub struct ValidationParams<'a> {
```

Comment:
Do we even still need this at all?

Reply:
probably not, we could just take a protocol parameters directly